skabase/SKALogger/SKALogger.py

```diff
@@ -190,12 +190,16 @@ def SetCentralLoggingLevel(self, argin):
         central_logging_device = argin[1][:]
         i = 0
         while i < len(central_logging_level[:]):
-            self.info_stream("Central Logging level : %s, Device : %s",
-                             central_logging_level[i],
-                             central_logging_level[i])
-            dev_proxy = DeviceProxy(central_logging_device[i])
-            dev_proxy.centralLoggingLevel = central_logging_level[i]
-            i += 1
+            try:
+                self.info_stream("Central Logging level : %s, Device : %s",
+                                 central_logging_level[i],
+                                 central_logging_level[i])
+                dev_proxy = DeviceProxy(central_logging_device[i])
+                dev_proxy.centralLoggingLevel = central_logging_level[i]
+                i += 1
```

Comment:
Having this inside the try..except, combined with not re-raising an exception on `DevFailed`, results in an infinite loop if we do get `DevFailed`.

Reply:
Corrected. Commit id 35e9e42 addresses this.